compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/GrpcServicePrinter.scala

```diff
@@ -175,6 +175,34 @@ final class GrpcServicePrinter(service: ServiceDescriptor, override val params:
           |""")
   }
 
+  private[this] def serviceDescriptor(service: ServiceDescriptor) = {
+
+    val grpcServiceDescriptor = "_root_.io.grpc.ServiceDescriptor"
+
+    val schemaDescriptor: String =
+      s""".setSchemaDescriptor(new _root_.io.grpc.protobuf.ProtoFileDescriptorSupplier {
+         |  override def getFileDescriptor: _root_.com.google.protobuf.Descriptors.FileDescriptor =
+         |    ${service.getFile.fileDescriptorObjectFullName}.javaDescriptor
+         |})"""
+
+    def addMethod(method: MethodDescriptor): PrinterEndo = PrinterEndo(
+      _.add(s".addMethod(${method.descriptorName})")
+    )
+
+    PrinterEndo(
+      _.add(s"val ${service.descriptorName}: $grpcServiceDescriptor =")
+        .indent
+        .add(s"""$grpcServiceDescriptor.newBuilder("${service.getFullName}")""")
+        .indent
+        .addStringMargin(schemaDescriptor)
```

Comment:
Given the previous comment, this may be just a simple `add()`. If not implement using `call()`